lib/rubocop/rspec/language/node_pattern.rb

```diff
@@ -23,6 +23,9 @@ module NodePattern
 
         def_node_matcher :let?, Helpers::ALL.block_or_block_pass_pattern
 
+        def_node_matcher :include?,
+                         Includes::ALL.send_or_block_or_block_pass_pattern
+
         def_node_matcher :subject?, Subject::ALL.block_pattern
```

Comment:
Side note.
Wondering if we should add support for proper detection of:
```ruby
before(:all, &global)
describe(&block)
shared_examples_for('foo', &block)
```

I dislike this syntax, but we've started supporting it step by step.

I'm thinking of a NoBlockPass cop along the way XD